config/env_sample.json

```diff
@@ -225,7 +225,7 @@
         "/*  LRS database port": "*/",
         "PORT": 5432,
         "/*  LRS database options": "*/",
-        "OPTIONS": {},
+        "OPTIONS": {}
```

Comment:
This looks Good, since you are there can you fix this as well
![Screen Shot 2021-02-04 at 3 48 28 PM](https://user-images.githubusercontent.com/8579775/106953506-7bd2b700-6700-11eb-9dfc-6ad55786b93c.png)

The port needs to be in same order as Host

Reply:
Yup, fixed. I just noticed this as I was trying to use this config and it wasn't flagging the line as an error.